Certification-Standard/introduction.md

```diff
@@ -0,0 +1,239 @@
+
+## 3.2 Target of Evaluation
+
+### 3.2.1 General
+
+The CCS should be flexible enough in what is evaluated so that it can be applied to a wide range of DApps. The CCS should be able to be applied to the whole DApp or only to a the most critical parts of it.
+
+The TOE could be a DApp, a part of a DApp, a set of full DApps, or a set of parts of DApps.
+
+One aspect that should be considered is the dependencies with external libraries. 
+
+The evaluation of a TOE containing only a part of a DApp should not be misrepresentative as the evaluation of a whole DApp.
+The evaluation of a TOE containing only a part of a DApp should be able to be used for the future evaluation of the whole DApp.
+
+## 3.2.2 TOE boundaries
+
+The concept of TOE boundary is fundamental to the specification of the ST.
+
+A TOE may be a complete DApp, a part of a DApp, a set of DApps, or a set of parts of DApps. The ST needs to clearly identify the logical boundaries of the TOE.
+
+Any parts of the overall product that are not included in the TOE are considered to be outside the scope of the evaluation.
+
+## 3.2.3 Parametrization of a TOE
+
+A DApp could be parametrized in many ways prior to deployment. Parts of a DApp could be created on-the-fly by the DApp itself. The TOE should clearly list the set of parameters that were considered.
+
+An evaluation of such a TOE should clearly list the set of parameters that were considered and on which the evaluation was based on. A certificate can only refers to a version of the TOE with a parameter choice that matches one in the set considered for the evaluation.
+
+## 3.2.4 Operational environment of the TOE
+
+Everything outside the TOE boundary belongs the Operational Environment (OE) of the TOE. In the case of the evaluation of part of DApp, the non-TOE parts of the DApp are parts of the OE.
+
+The ST needs to describe the assumptions on the OE so that together with the security functionalities provided by the TOE, the security objectives are met.
+
+## 3.3 Specifying Security Requirements
+
+### 3.3.1 Security Problem Definition
+
+The SPD defines the security problem that is to be adressed and is presented in the ST. The SPD defines the threats that are to be countered by the TOE, the OE assumptions, and the organizational security policies.
+
+#### Threats
+
+The SPD describes the threats that the TOE is intended to counter.
+
+A threat is an adverse action that can be performed by a threat agent on an asset.
+
+## 3.4 Security Target
+
+A ST is a document that describes:
+- a specific TOE (features, life cycle, interfaces, the intended usage, ...)
+- Conformance Claims (Which version of the standard, which PP, which EAL, ...)
+- Security problem definition (assets, threats, organizational security policies, assumptions, ...)
+- Security objectives (functional and assurance)
```

Comment:
what's the purpose of "functional and assurance" here?

Reply:
Roughly, the functional requirements are what the TOE should do. The assurance requirements are how it should be tested, what activities should be done to ensure that it is compliant.

For example, the assurance requirements for a DApp could say: I want my TOE to be evaluated for a Level 3 certification. This would imply some activities for the developer had to perform some kind activities in the definition of  the formal model for example and provide some results/documentation (like proofs, assumptions, solver used etc.)

Reply:
But it might be too much for the moment, and maybe we can focus on functional requirements for now